### ISSUE TITLE
Langflow UI stuck in Building without actually building the components 

### ISSUE THREAD

**Issue by ArianeFire** [langflow-ai/langflow#1968]
**Describe the bug**
Hello and thanks for the amazing work,
Langflow regularly stuck in building state until I restart it like shown in the attached image.

Can someone help or is it a bug ?

**Browser and Version**
 - Browser chrome

**To Reproduce**
Work on langflow for sometime (Editing flow, Running some components, ...)
Leave the browser window open for sometime (Some minutes)
Refresh or switch to another flow 
Try to run a flow it get stuck in building.... but the components never run

**Screenshots**
<img width="1792" alt="FREEZE_BUILDING" src="https://github.com/langflow-ai/langflow/assets/12550052/b0fbf6be-09ce-48bf-a7f6-ad1ac6b1b0a5">

**Additional context**
When I restart, It work fine but there's nothing show on the logs and the the building Icon is not animated on the components

**Comment by AndrewSun1010** [comment]
in the same situation, have you fixed it bro?

**Comment by ArianeFire** [comment]
No, it's a blocking issue for production, can't go to production with this. 

I am suspecting the cache management to be the cause, One of my flow blocked forever just after a cache expiration message popup when running it through json

After adding the below code to extends the cache expiration time (Not proper way), the flow run properly (I will be testing it on langflow web to see) : 

```
from langflow.services.deps import get_cache_service
et = get_cache_service().expiration_time
get_cache_service().expiration_time = et * et * et
```

**Comment by AndrewSun1010** [comment]
i fixed my problem, check in #2218
